### PR TITLE
Fixes #32721 - Add GraphQL queries for JobInvocations

### PR DESCRIPTION
**Pull request by xprazak2**
Needs https://github.com/theforeman/foreman-tasks/pull/634 as it references the types defined there.

### CODE REVIEW

File: app/graphql/types/job_invocation.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Types
+  class JobInvocation < BaseObject
+    description 'A Job Invocation'
+
+    global_id_field :id
+    field :job_category, String
+    field :description, String
+    field :time_span, Integer
+    field :start_at, GraphQL::Types::ISO8601DateTime
+    field :status_label, String
```
Comment (adamruzicka) [inline app/graphql/types/job_invocation.rb:10]: What is status label?

Comment (xprazak2) [reply app/graphql/types/job_invocation.rb:10]: https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/job_invocation.rb#L134

status-label

Comment (adamruzicka) [reply app/graphql/types/job_invocation.rb:10]: Ah, I completely forgot we have this thing. Thank you for reminding me

**Comment by adamruzicka** [comment]
theforeman/foreman-tasks#634 just got merged and will be released in foreman-tasks 5.0.0. Could you bump the dependency here?

**Comment by xprazak2** [comment]
Done, will not bundle until foreman-tasks are released.

**Comment by adamruzicka** [comment]
Tasks 5.0.0 are out, could you force push so all the tests start running again?

**Comment by xprazak2** [comment]
Rebased.

**Comment by xprazak2** [comment]
Are there any additional changes I should make here?

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @xprazak2 !

**Comment by MariaAga** [comment]
I'm trying to run `foreman s` and I get this error:
```
 /home/vagrant/foreman_remote_execution/app/graphql/types/job_invocation.rb:12:in `<class:JobInvocation>': uninitialized constant Types::Triggering (NameError)
```
and then `SIGTERM` ):
@adamruzicka @xprazak2

**Comment by adamruzicka** [comment]
Do you have latest foreman-tasks?

**Comment by MariaAga** [comment]
Yes, also this file is only added\changed in this pr

**Comment by adamruzicka** [comment]
Could you try `bundle exec spring stop` and then try `foreman s`?

**Comment by MariaAga** [comment]
Didn't help

**Pull request merged**